docs/file-map-and-description.md

```diff
@@ -24,10 +24,11 @@ File descriptions for template
   [Most popular open-source licenses](https://opensource.org/licenses).
 - [README.md](../README.md) - main readme for repository.
 - [docs](../docs) - documentation for repository.
-- [requirements.txt](../requirements.txt) - list of required 3rd party packages to run your project.
+<!-- - [requirements.txt](../requirements.txt) - list of required 3rd party packages to run your project.
 - [requirements-dev.txt](../requirements-dev.txt) - list of required 3rd party packages that are
   NOT required to run your project, but which might benefit developers. It can include specific test
-  libraries, style checks packages etc.
+  libraries, style checks packages etc. -->
```

Comment:
Can you remove these lines containing `requirements.txt` and `requirements-dev.txt`? (since there is no chance in them coming back)

Reply:
That has been resolved.

Reply:
Can you double check?  I think you removed a different commented region, actually, than this one.  I was thinking it makes sense to keep the explanations in `README.md` commented out since (we hope) the development and minimum version tests will re-appear in a subsequent PR.

Reply:
Hello @garrison you can check my last push https://github.com/qiskit-community/quantum-prototype-template/pull/70/commits/a4a72f7d37dd995cac8bdb20e1b742e0c58cbfb6 it contains the changes you requested.

Reply:
I have also re-added the comment section I wrongly removed previously. Sorry about that.